src/code/reducers/modal-dialog.js

```diff
@@ -25,7 +25,8 @@ export default function modalDialog(state = initialState, action) {
     case actionTypes.TOGGLE_MAP:
       return state.merge({
         show: action.isVisible,
-        showMap: action.isVisible
+        showMap: action.isVisible,
+        showAward: false
```

Comment:
True, but we certainly wouldn't want to show award just because the user toggled the map off.

I guess we could imagine that the user was looking at the award, went to the map, and then wanted to look at the award again. But then we'd have to save some temporary state to say that they had been looking at the award. I'm not sure that it's necessary, though. I think if you toggle the "map" off, you expect the whole widget to go away.